source/jsonnet/northern-ireland/household/blocks/who-lives-here/people_living_here.jsonnet

```diff
@@ -102,7 +102,7 @@ local editQuestion(questionTitle) = {
 };
 
 {
-  id: 'anyone-else-list-collector',
+  id: 'people-living-here',
```

Comment:
The rest of the ids should be made consistent with this.

Reply:
Other ids added. I followed ENG schemas examples. add/edit/remove-person ids has to stay the same. Thanks!👍